apps/survey/.env-template

```diff
@@ -40,6 +40,9 @@ VITE_CAPTCHA_SITEKEY=
 VITE_GOOGLE_ANALYTICS_ID=
```

Comment:
Can you put GA and GTM IDs close together so both tracking IDs are close, will be easier to navigate.